client/components/Search.js

```diff
@@ -1,35 +1,147 @@
 import React from 'react'
 import elasticSearchClient from '../services/ElasticSearch'
+import Searchbar from './Searchbar'
+import SearchResult from './SearchResult'
 const { string } = React.PropTypes
+const numberOfResults = 12
 
-const Search = React.createClass({
-  propTypes: {
-    searchTerm: string
-  },
-  getInitialState () {
+export default class Search extends React.Component {
```

Comment:
ES6 classes are totally okay(and probably better) but couldn't understand why this component is an ES6 class and the other one is React.createClass({}).